woocommerce/google-listings-and-ads
# Fix related products E2Es for WC9.5

### Changes proposed in this Pull Request:



This PR adjusts E2E tests for WC 9.5 changes.

In WC 9.5, by default, there is no related products section on a single product page - I made the test skip in such a case.
Also, I added a "related" tag to all simple products to make sure simple products will appear in the related products section instead of "Cat teasers" - without the "Add to cart" button.



### Screenshots:

![image](https://github.com/user-attachments/assets/7c9a774c-2b82-4421-8184-0c014d0434d2)





### Detailed test instructions:


1. Run E2E tests locally
2. Run [E2E workflow](https://github.com/woocommerce/google-listings-and-ads/actions/workflows/e2e-tests.yml) with `9.5.0-rc.1` - https://github.com/woocommerce/google-listings-and-ads/actions/runs/12361402622
3. Run [E2E workflow](https://github.com/woocommerce/google-listings-and-ads/actions/workflows/e2e-tests.yml) with `9.4.0` - https://github.com/woocommerce/google-listings-and-ads/actions/runs/12361396232
4. Review the code to check the tests are testing what they should


### Additional details:


### Changelog entry

> Dev - Improve E2E tests for related products.
